unstructured/partition/pptx.py

```diff
@@ -53,11 +53,9 @@ def partition_pptx(
     file: Optional[IO[bytes]] = None,
     include_page_breaks: bool = True,
     metadata_filename: Optional[str] = None,
-    include_metadata: bool = True,
```

Comment:
Do we need to create a follow-up issue to add this as a functioning parameter?

Reply:
No. I asked Matt about it and it is not a published (end-user) option. It has something to do with file-conversion he said so if we're not doing anything with it in a particular partitioner we can just let it get collected by `kwargs` like other options the particular partitioner is insensitive to.